app/store/aliases.ts

```diff
@@ -0,0 +1,27 @@
+import * as axios from "axios";
+import {fetchSuccess, fetchFailure} from "./../actions";
+
+const fetchDataAlias = (payload: object) => {
+  return dispatch => {
+    const {method, url, requestId, requestHeaders} = payload.requestDetails;
+    const requestHeadersObject = requestHeaders.reduce((accumulatedObj, reqHeaderNameValuePair) => {
+      accumulatedObj[reqHeaderNameValuePair.name] = reqHeaderNameValuePair.value;
+      return accumulatedObj;
+    }, {});
+    axios({
+      method,
+      url,
+      requestHeadersObject
+    })
+      .then(data => {
+        dispatch(fetchSuccess(data, requestId));
+      })
+      .catch(err => {
+        dispatch(fetchFailure(err, requestId));
+      });
+  };
+};
+
+export default {
+  FETCH_DATA: fetchDataAlias
```

Comment:
please don't do this. 
please `export default fetchDataAlias`

Reply:
This is confusing can we use the normal thing since we are not integrating anything with background.